frontend/src/AudioAnnotator/Workbench.js

```diff
@@ -425,7 +426,10 @@ class Workbench extends Component<WorkbenchProps, WorkbenchState> {
     const newAnnotation: Annotation = {
       type: TYPE_BOX,
       id: '',
-      annotation: '',
+      annotation:
+      typeof this.props.currentDefaultTagAnnotation !== "undefined"
+        ? this.props.currentDefaultTagAnnotation
+        : '',
```

Comment:
This feels very off to me, using `typeof VAR !== "undefined"` seems like it should be a last resort in JS :thinking:.

Also I'm not at all sure this is necessary, in frontend/src/AudioAnnotator/AudioAnnotator.js line 131 you define the default value as "" and this should be propagated all the way here : I don't think `props` can be `undefined` (here in particular but also in react in general).

I've just tested your code with just `annotation: this.props.currentDefaultTagAnnotation` and it works fine. Did you jump the gun here or was it something that was necessary on an older version of the code ?